.env.test

```diff
@@ -0,0 +1,6 @@
+ DB_PASSWORD=password
+ DB_NAME=music_library_test
+ DB_USER=user
+ DB_HOST=localhost
+ DB_PORT=3308
+ PORT=3000
```

Comment:
Not related to this file, but I couldnt comment on an empty one! The .DS_Store file shouldnt be included here